netlify/functions/manage-findings.ts

```diff
@@ -210,32 +251,36 @@ async function editFinding(
 ): Promise<void> {
   const CustOcto = Octokit.plugin(createOrUpdateTextFile);
   const client = new CustOcto({ auth: process.env.GITHUB_TOKEN });
-  if(data.mitigationOf?.oldValue !== data.mitigationOf?.newValue){
+  if (data.mitigationOf?.oldValue !== data.mitigationOf?.newValue) {
     throw {
       message: `You cannot change the report id ${data.mitigationOf?.oldValue} for this mitigation issue.`,
     };
   }
- 
+
   //is user marked issue as mitigation confirmed but then changed their minds remove the mitigation confirmed label and add new risk label and body
   //mitigation confirmed and no risk label exists
-  let oldRiskCode = '';
-  let newRiskCode = '';
-  if(!data.mitigationOf){
+  let oldRiskCode = "";
+  let newRiskCode = "";
+  if (!data.mitigationOf) {
     oldRiskCode = getRiskCodeFromGithubLabel(data.risk.oldValue);
     newRiskCode = getRiskCodeFromGithubLabel(data.risk.newValue);
-  }else{
-    oldRiskCode = data.risk.oldValue ? getRiskCodeFromGithubLabel(data.risk.oldValue) : '';
-    newRiskCode = data.risk.newValue ? getRiskCodeFromGithubLabel(data.risk.newValue) : '';
+  } else {
+    oldRiskCode = data.risk.oldValue
```

Comment:
I thought we had already caught each repo up with this sort of linting?

Reply:
Maybe not this one? I don't know. I just auto lint on save.

Reply:
yep, made the PR harder to review but not a blocker.